python/ctsm/git_utils.py

```diff
@@ -0,0 +1,66 @@
+"""General-purpose git utility functions"""
+
+import logging
+import subprocess
+from ctsm.path_utils import path_to_ctsm_root
+
+logger = logging.getLogger(__name__)
+
+
+def get_git_short_hash():
+    """
+    Returns Git short SHA for the currect directory.
```

Comment:
I agree that it would be more clear to change them to something that has "ctsm_git" in it.

Reply:
I changed the names of these functions to:

`get_git_short_hash` -> `get_ctsm_git_short_hash`
`get_git_long_hash` -> `get_ctsm_git_long_hash`
`get_git_describe` -> `get_ctsm_git_describe`

Is that good?
I think it is a good idea but it might be a bit long name for a simple functions but it is okay.

Reply:
I like the new names. We sometimes worry about names being too long, but the main importance is clarity, and I think the new names are more clear. Usually the only problem with long names (even for simple things) is it's harder to write the first time. But, we read code 10X or more than the amount of time we write it, so length really isn't a problem. You can have long names that are confusing though, because they are too long. Or you if you have expressions with long names that can become unwieldy and hard to read. But, I can't say I've seen that as a problem hardly at all, usually names can be longer and more descriptive.